src/miner.cpp

```diff
@@ -312,35 +308,32 @@ CBlockTemplate* CreateNewBlock(const CChainParams& chainparams, const MinerAddre
 
                 int nConf = nHeight - coins->nHeight;
 
-                dPriority += (double)nValueIn * nConf;
             }
             nTotalIn += tx.GetShieldedValueIn();
 
             if (fMissingInputs) continue;
 
-            // Priority is sum(valuein * age) / modified_txsize
             unsigned int nTxSize = ::GetSerializeSize(tx, SER_NETWORK, PROTOCOL_VERSION);
-            dPriority = tx.ComputePriority(dPriority, nTxSize);
 
+            double nDummyPriority = 0;
             uint256 hash = tx.GetHash();
-            mempool.ApplyDeltas(hash, dPriority, nTotalIn);
+            mempool.ApplyDeltas(hash, nDummyPriority, nTotalIn);
 
             CFeeRate feeRate(nTotalIn-tx.GetValueOut(), nTxSize);
 
             if (porphan)
             {
-                porphan->dPriority = dPriority;
                 porphan->feeRate = feeRate;
             }
             else
-                vecPriority.push_back(TxPriority(dPriority, feeRate, &(mi->GetTx())));
+                vecPriority.push_back(TxPriority(nDummyPriority, feeRate, &(mi->GetTx())));
```

Comment:
Same thing here, we should remove priority entirely.

Reply:
done!